artemis/modules/classifier.py

```diff
@@ -39,6 +41,14 @@ def is_supported(data: str) -> bool:
         if Classifier._to_ip_range(data):
             return True
 
+        if ":" in data:
+            data, port = data.split(":", 1)
```

Comment:
netloc scheme also allows for user auth in URI - unlikely, but still this should support it:
```suggestion
            data, port = data.rsplit(":", 1)
```